js/src/lib/api/track.ts

```diff
@@ -39,3 +39,20 @@ interface HistoryActionProps {
 export function trackHistoryAction(props: HistoryActionProps) {
   amplitude.track("[Web] history_action", props);
 }
+
+interface PreviewChangeProps {
+  action: "explore" | "run";
```

Comment:
Should we record a `close` event?

Reply:
yes, we should record user who just leave without action